composer.json

```diff
@@ -29,7 +29,7 @@
         "mimmi20/file-loader": "~2.0",
         "symfony/filesystem": "*",
         "symfony/finder": "~2.6",
-        "symfony/console": "~2.6",
+        "symfony/console": "^3.0",
```

Comment:
@ThaDafinser Is there something I need to change for that or is it an option in Travis that guys need to update?